lib/pymedphys/docs/conf.py

```diff
@@ -15,12 +15,8 @@
 
 # pylint: disable = C0413, C0103
 
-import os
-
 import pymedphys
 
-os.system("pandoc CHANGELOG.md --from markdown --to rst -s -o release-notes.rst")
```

Comment:
paired with introduction of this as check_process above in this PR.
the pair probably should have been done in a separate PR

Reply:
Yup see https://github.com/pymedphys/pymedphys/pull/1169 I cherry-picked that commit into a separate PR.